src/widget/wmainmenubar.cpp

```diff
@@ -532,6 +530,22 @@ void WMainMenuBar::initialize() {
 
     QString externalLinkSuffix = " =>";
 
+    //: menu title
+    QString keywheelTitle = tr("Show Keywheel");
+    //: tooltip text
+    QString keywheelText = tr("Show keywheel");
+    m_pViewKeywheel = new QAction(keywheelTitle, this);
+    m_pViewKeywheel->setCheckable(true);
+    m_pViewKeywheel->setShortcut(
+            QKeySequence(m_pKbdConfig->getValue(
+                    ConfigKey("[KeyboardShortcuts]", "ViewMenu_ShowKeywheel"),
+                    tr("F2", "Menubar|View|Show Keywheel"))));
```

Comment:
This shortcut clashes with the existing shortcut for `[Channel1],rate_perm_up`, see https://github.com/mixxxdj/mixxx/blob/main/res/keyboard/en_US.kbd.cfg#L43